src/prelude.rs

```diff
@@ -51,7 +51,7 @@ pub use auto::traits::ToolItemExt;
 pub use auto::traits::ToolShellExt;
 pub use auto::traits::TreeModelExt;
 //pub use auto::traits::TreeSortableExt;
-//pub use auto::traits::WidgetExt;
+pub use auto::traits::WidgetExt as WidgetExtGenerated;
```

Comment:
Without exporting `auto::traits::WidgetExt` build fails.
Currently we can't export `auto::traits::*` due we still hide `auto::traits::TreeSortableExt`.
Or not?